features/x-provenance.rst

```diff
@@ -10,9 +10,25 @@ The X-Provenance header can be used to add a Provenance resource upon creating o
 
 .. note:: In the case of a conditional create, where the resource was already present, the Provenance resource is not created.
 
+Linking the Provenance resource
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Provenance must point to it's target resource through the ``target`` property (see `Provenance#resource <https://www.hl7.org/fhir/Provenance.html#resource>`_)
 The ``target`` will be automatically deduced from the created/updated resource and does not need to be included in the header. 
 The reference will point to the current version (after create/update) of the resource.
 
+Validation of the Provenance resource
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Validation of the Provenance resource follows the settings for validation (see :ref:`feature_prevalidation`). 
+
+.. note:: If validation is disabled, an invalid Provenance resource in the header will be ignored and the request will complete normally (if valid).
+
+Limitations
```

Comment:
I think that’s self explanatory

Reply:
Hmm, well maybe the phrasing is not accurate. I want to express that, if you don't have validation on, you'll not notice that your Provenance resource is not added (which I think is not exactly the same as when you submit an invalid Patient resource like {}).

Reply:
I removed the sentence